src/createLanguageClient.ts

```diff
@@ -160,8 +190,6 @@ function createLanguageClient (
     connectionProvider: new CGLSPConnectionProvider(id, infrastructure)
   })
 
-  client.registerProposedFeatures()
```

Comment:
what was this about ?

Reply:
There is no proposed feature anymore, this method is empty now

Reply:
It is about `beta` features not already present in the protocol